english/explore-outside/talk-child/talk-more/nice-talk/help-child/keep-looking/enter-cave/follow-person/follow-person.md

```diff
@@ -1,4 +1,14 @@
 You use all your strength to swim toward the light. The person ahead swam out of sight. Can you make it?
 Yes! When your strength was almost spent you realized that it was getting shallow and you could stand up.
 There is an air space above your head! You can breathe!
-Taking in great heaving gulps of sweet air (well, kind of damp cave air) you...
+Taking in great heaving gulps of sweet air (well, kind of damp cave air) you take stock of your surroundings.
+
+You are in a vast cavern with walls covered in tiny glittering crystals. A wan purple light from somewhere above reflects off the crystal and the yellow water. In front of you, the green child glares at you fearfully. His Pittsburgh accent is gone.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.